### DISCUSSION TITLE
can i use this for wifi direct?

### DISCUSSION THREAD

**Discussion by hmmhmmhm** [chris1111/Wireless-USB-Big-Sur-Adapter#146] [Q&A] [answered]
I'm asking just curious.

**Comment by chris1111** [comment, marked as answer]
Wifi Direct?

**Reply by hmmhmmhm** [reply]
I was wondering if it would be possible to create some internal network with usb wifi in the mac book.
https://blog.immersed.team/wi-fi-direct-8ec23c74fdab

**Reply by hmmhmmhm** [reply]
![스크린샷 2021-10-03 오후 10 05 41](https://user-images.githubusercontent.com/38493722/135754911-627dc7e7-2347-4a6d-a22b-a57428095e2e.png)
![스크린샷 2021-10-03 오후 10 05 33](https://user-images.githubusercontent.com/38493722/135754923-ac03a0ae-f63c-497c-9f0e-47ae81d20ca0.png)
![스크린샷 2021-10-03 오후 10 06 40](https://user-images.githubusercontent.com/38493722/135754941-879934b9-6569-40f6-999d-f053c2c3a9b7.png)

I actually tested it, but i don't think mac internet sharing options doesn't works.
I don't know if I did it right. I turned off the SIP,
entered the command well, and the simple Wi-Fi connection works well.
Only internet sharing doesn't work...

**Reply by hmmhmmhm** [reply]
OH my gosh... i'm sorry never mind...
It works well with connect Wi-Fi through a program other than the existing Mac Wi-Fi function.
Thank you sir for making a good program.

![스크린샷 2021-10-03 오후 10 16 07](https://user-images.githubusercontent.com/38493722/135755348-059cea5a-821c-418e-beb2-c072dac011ba.png)
![스크린샷 2021-10-03 오후 10 16 11](https://user-images.githubusercontent.com/38493722/135755353-646848b2-6f2e-41bf-928a-0ea42c103a56.png)
